### PR TITLE
bld.bat file is added for creating bwidget package in a windows conda…

### PR DESCRIPTION
**Pull request by nandandevinagara**
… environment.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by abloemert** [comment]
@conda-forge-admin, please rerender

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,5 @@
+set TCL_LIB_PATH=%PREFIX%\Library\lib\tcl8.6\%PKG_NAME%%PKG_VERSION%
+mkdir %TCL_LIB_PATH%
```
Comment (fhoehle) [inline recipe/bld.bat]: Please add a safety net checking if the directory exists. Or add a line raising errors: `if errorlevel 1 exit 1` see here https://github.com/conda-forge/r-base-feedstock/blob/19c643d97f43050424be0686a4d22dd88fcc5e36/recipe/bld-r-base.bat#L18

the following lines could also need this precaution.

**Review by fhoehle — CHANGES_REQUESTED** [review]
Thanks for adding windows builds. I would like to have some error checking statements in `bld.bat` to ensure potential errors are raised. When this PR is merged, do you plan switch to this `bwiget` build in `r-base`? See here: https://github.com/conda-forge/r-base-feedstock/blob/e74584ad023e552247fc9451a6ce2e576be2b439/recipe/meta.yaml#L136

File: recipe/bld.bat (hunk 1 above)
Comment (nandandevinagara) [reply recipe/bld.bat]: Thanks for the hint @fhoehle. I will add the error-checking commands. 
I didn't plan to switch to bwidget build in r-base.

Comment (nandandevinagara) [reply recipe/bld.bat]: I have implemented the necessary changes. Could you please review it, @fhoehle

Comment (fhoehle) [reply recipe/bld.bat]: Thanks adding

**Review by fhoehle — APPROVED** [review]

**Pull request merged**